src/app/sidenav/sidenav.component.unit.spec.ts

```diff
@@ -234,116 +204,4 @@ describe('MzSidenavComponent:unit', () => {
       expect(mockCollapsible.collapsible).toHaveBeenCalled();
     });
   });
-
-  describe('fixCloseOnClick', () => {
-
-    function createSidenavLink(collapsible?: boolean): void {
-      const link = document.createElement('a');
-      if (collapsible) {
-        link.classList.add('collapsible-header');
-      }
-      const header = document.createElement('li');
-      header.appendChild(link);
-      $(fixture.nativeElement).find('.side-nav').append(header);
-    }
-
-    describe('non-collapsible link', () => {
-      let renderedLink: HTMLElement;
-      const mockCollapseButton = { sideNav: (method: string) => {} };
-      const mockWindow = { width: () => 0 };
-
-      beforeEach(() => {
-
-        createCollapseButton(collapseButtonId);
-        component.collapseButtonId = collapseButtonId;
-
-        createSidenavLink();
-
-        component.fixCloseOnClick();
-
-        renderedLink = $(fixture.nativeElement).find('li a')[0];
-
-        spyOn(mockCollapseButton, 'sideNav');
-
-        spyOn(window, '$').and.callFake((selector: string): any => {
-          return selector === `#${collapseButtonId}`
-            ? mockCollapseButton
-            : mockWindow;
-        });
-      });
-
-      it('should hide sidenav on click when screen width < 992px', () => {
-
-        spyOn(mockWindow, 'width').and.returnValue(991);
-
-        renderedLink.click();
-
-        expect(mockCollapseButton.sideNav).toHaveBeenCalledWith('hide');
-      });
-
-      it('should not hide sidenav on click when screen width >= 992px', () => {
-
-        spyOn(mockWindow, 'width').and.returnValue(992);
-
-        renderedLink.click();
-
-        expect(mockCollapseButton.sideNav).not.toHaveBeenCalled();
-      });
-
-      it('should keep track of listeners', () => {
-
-        expect(component.closeOnClickListeners.length).toBe(1);
-      });
-    });
-
-    describe('collapsible link', () => {
```

Comment:
Why do we remove those unit tests?

Reply:
The test was meant to be sure that `fixCloseOnClick` won't close the menu on mobile when clicking on a item that should collapse (not a page link)